### PR TITLE
 Add profile header information querying and updated event query

### PR DESCRIPTION
**Pull request by tsunamit**

### CODE REVIEW

File: src/resolvers/Mutation.js [hunk 1]
```diff
@@ -231,6 +231,21 @@ async function updateEventStatus(parent, { eventId, currentEventStatus, newEvent
   });
 }
 
+async function updateProfileDetails(parent, { name, bio }, context) {
+  const userId = await getUserId({ context });
+
+  const data = {};
+  if (name != null) data.name = name;
+  if (bio != null) data.bio = bio;
```
Comment (ajberdy) [inline src/resolvers/Mutation.js]:
```suggestion
  const data = { name, bio };
```

File: src/resolvers/User.js [hunk 2]
```diff
@@ -76,7 +65,11 @@ async function events(parent, { association, status }, context) {
   return eventList;
 }
 
+async function ownedEvents(parent, args, context) {
+  return events(parent, { association: ['OWNER'], status: ['SET', 'INVITED', 'JOINED'] }, context);
+}
+
```
Comment (ajberdy) [inline src/resolvers/User.js]: I like this

File: prisma/generated/prisma-client/prisma-schema.js [hunk 3]
```diff
@@ -1133,6 +1181,28 @@ input UserWhereInput {
...
+  bio: String
+  bio_not: String
+  bio_in: [String!]
+  bio_not_in: [String!]
+  bio_lt: String
+  bio_lte: String
+  bio_gt: String
+  bio_gte: String
+  bio_contains: String
+  bio_not_contains: String
```
Comment (ajberdy) [inline prisma/generated/prisma-client/prisma-schema.js]:
```suggestion
  bio_gte: Int
  bio_contains: Int
  bio_not_contains: Int
```

File: src/resolvers/User.js [hunk 4]
```diff
@@ -1,16 +1,5 @@
...
-    where: {
-      owner: {
-        id: parent.id,
-      },
-    },
-  });
-}
-
-// eslint-disable-next-line no-unused-vars
 async function events(parent, { association, status }, context) {
```
[the remark this answers is not in the document]
Comment (tsunamit) [reply src/resolvers/User.js]: done

**Pull request merged**